docs/articles/apex/testing.md

```diff
@@ -19,7 +19,7 @@ To clear your test results, click the refresh icon at the top of the sidebar (ho
 ## Run Apex Tests from Within a File
 
 To run Apex tests, in your `.cls` file, click **Run Test** or **Run All Tests** above the definition of an Apex test method or class.  
-![Running Apex tests using the Run Test and Run All Tests code lenses](https://raw.githubusercontent.com/forcedotcom/salesforcedx-vscode/develop/packages/salesforcedx-vscode-core/images/apex_test_run.gif)
+![Running Apex tests using the Run Test and Run All Tests code lenses](../../images/apex_test_run.gif)
```

Comment:
Is changing the references to relative paths fixing an issue with running the doc site locally ?

Reply:
Locally and on netlify. If you need to change the image or add a new image it doesnt work if its hosted statically. The preview site should show the updated image.